src/jit/compiler.h

```diff
@@ -6931,6 +6929,9 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
     // Should we support SIMD intrinsics?
     bool featureSIMD;
 
+    // Have we identified any SIMD types?
+    bool foundSIMDType;
```

Comment:
I actually prefer the comment above the declaration.
I've changed the references to accessors (though I'm using setUsesSIMDTypes(bool value).

---
In reply to: [95666422](https://github.com/dotnet/coreclr/pull/8402#discussion_r95666422) [](ancestors = 95666422)